m4_sessions/1706-inning/group_five.md

```diff
@@ -1,9 +1,50 @@
 # Group Five
 
-### Session Title:
+### Session Title: How and When to Learn a New Language?
 
 ### Session Outcomes: 
+- Create a repo of resources and tutorials for learning new languages and technologies that students can contribute to.
+- Provide outline of our process and experiences
+- Provide space for people to share their own challenges and solutions.
 
 ### Session Outline:
 
-### Names of group members:
+#### Whole Group discussion
+
+##### 10 minute introduction, 
+
+##### 15 min intro
+
+
+- Deliberate Practice
+  - Process of learning
+    - ID a specific subset
+    - Focus on performance in that subset
+    - Get feedback
+    - Update 
+
+   - Motivations
+    - Project
+    - Teaching deadline
+   - Pitfalls
+     - Don't read book cover to cover without coding along the way.
+      
+- Movie: TBD
+
+- Where would you start with a new programming language? /describe our experiences
+  - Docs
+  - Books
+  - Stack Overflow
+  - Find Online communities
+  
+
+##### 30 min dig in
```

Comment:
Why 5 groups?  I might give suggested times for each step of the process and maybe have something specific for each group to share out.